### PR TITLE
Harmonize raster "stack position", "frequency" and "cell statistics"  algorithms description

### PR DESCRIPTION
**Pull request by DelazJ**
These algs seem to have a bunch in common but their help string is written differently, leading to some missing information for some, and more work on translators. Using as far as possible common strings makes description complete, translation process easier. Main changes:
- re frequency alg, I suggest to rename the "input value" to "reference value" (i know this is how arc*** calls it) but imho makes referring to "input layer" in the GUI less confusing. Also it mimics the other algs. 
- I assume that resampling is done whenever the raster are not aligned with the reference layer.
- avoid double negation: "ignore nodata is not checked"
- few typos here and there

### CODE REVIEW

File: src/analysis/processing/qgsalgorithmrasterfrequencybycomparisonoperator.cpp [hunk 1]
```diff
@@ -246,14 +246,15 @@ QString QgsRasterFrequencyByEqualOperatorAlgorithm::shortHelpString() const
 {
   return QObject::tr( "The Equal to frequency algorithm evaluates on a cell-by-cell basis the frequency "
                       "(number of times) the values of an input stack of rasters are equal "
-                      "to the value of a value raster. \n "
+                      "to the value of a reference raster layer. \n "
                       "If multiband rasters are used in the data raster stack, the algorithm will always "
-                      "perform the analysis on the first band of the rasters - use GDAL to use other bands in the analysis. "
-                      "The input value layer serves as reference layer for the sample layers. "
+                      "perform the analysis on the first band of the rasters - use GDAL to use other bands in the analysis.\n "
```
Comment (DelazJ) [inline src/analysis/processing/qgsalgorithmrasterfrequencybycomparisonoperator.cpp:251]: Any idea which GDAL commands/algs can be used here? We can point to them in the online documentation (https://github.com/qgis/QGIS-Documentation/pull/6872)

File: src/analysis/processing/qgsalgorithmcellstatistics.cpp [hunk 2]
```diff
@@ -398,11 +398,11 @@ QString QgsCellStatisticsPercentileAlgorithm::shortHelpString() const
                       "   <li>Exclusive linear interpolation (PERCENTILE.EXC)</li>"
                       "</ul> "
                       "While the output value can stay the same for the nearest rank method (obtains the value that is nearest to the "
-                      "specified percentile), the linear interpolation method return unique values for different percentiles. Both interpolation "
+                      "specified percentile), the linear interpolation methods return unique values for different percentiles. Both interpolation "
                       "methods follow their counterpart methods implemented by LibreOffice or Microsoft Excel. \n\n"
```
Comment (DelazJ) [inline src/analysis/processing/qgsalgorithmcellstatistics.cpp:402]: same here. Any refs to those implementation?

**Review by DelazJ — COMMENTED** [review]
Looking for references...

File: src/analysis/processing/qgsalgorithmrasterfrequencybycomparisonoperator.cpp (hunk 1 above)
Comment (root676) [reply src/analysis/processing/qgsalgorithmrasterfrequencybycomparisonoperator.cpp:251]: When I wrote the algorithm I was thinking about `gdal:translate` algorithm.

File: src/analysis/processing/qgsalgorithmcellstatistics.cpp (hunk 2 above)
Comment (root676) [reply src/analysis/processing/qgsalgorithmcellstatistics.cpp:402]: What do you mean by references? References to the specific implementations?

**Comment by root676** [comment]
Thanks for harmonizing the helpstrings!

File: src/analysis/processing/qgsalgorithmrasterfrequencybycomparisonoperator.cpp (hunk 1 above)
Comment (nyalldawson) [reply src/analysis/processing/qgsalgorithmrasterfrequencybycomparisonoperator.cpp:251]: Probably the "rearrange bands" algorithm is the correct one here

**Comment by nyalldawson** [comment]
@DelazJ what's the status here?

**Comment by DelazJ** [comment]
Sorry. Unfinished. I'd like to mention the rearrange bands alg and I need to double check some of my changes. Will try to do it in upcoming days

**Pull request closed without merging**